ViewModel/AlternativeComparisonReport/AlternativeComboItem.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ViewModel.Alternatives;
+
+namespace ViewModel.AlternativeComparisonReport
+{
+    public class AlternativeComboItem:BaseViewModel
+    {
+        private AlternativeElement _Alternative;
+        private string _Name;
+        public AlternativeElement Alternative
```

Comment:
I think you are asking me to do this:
 public AlternativeElement Alternative { get; } 
and get rid of the private field. If i do that, i will need a setter on the Alternative because it gets set in the update method of this class.

Reply:
i moved to one line.